host.go

```diff
@@ -0,0 +1,36 @@
+package provider
+
+type OtelConfig struct {
+	EnableObservability   bool   `json:"enable_observability"`
+	EnableTraces          *bool  `json:"enable_traces,omitempty"`
+	EnableMetrics         *bool  `json:"enable_metrics,omitempty"`
+	EnableLogs            *bool  `json:"enable_logs,omitempty"`
+	ObservabilityEndpoint string `json:"observability_endpoint,omitempty"`
+	TracesEndpoint        string `json:"traces_endpoint,omitempty"`
+	MetricsEndpoint       string `json:"metrics_endpoint,omitempty"`
+	LogsEndpoint          string `json:"logs_endpoint,omitempty"`
+}
+
+type HostData struct {
+	HostID             string `json:"host_id,omitempty"`
+	LatticeRPCPrefix   string `json:"lattice_rpc_prefix,omitempty"`
+	LatticeRPCUserJWT  string `json:"lattice_rpc_user_jwt,omitempty"`
+	LatticeRPCUserSeed string `json:"lattice_rpc_user_seed,omitempty"`
+	LatticeRPCURL      string `json:"lattice_rpc_url,omitempty"`
+	ProviderKey        string `json:"provider_key,omitempty"`
+	// NOTE: Rust is trying to pass a Vec<(String, String)> here, can we support that?
```

Comment:
We should probably do a custom `UnmarshalJSON` function to handle this for now

Reply:
I did test this actually, and it works just fine 😄 Even though right now in the host we always pass a `vec![]`. In rust, passing `vec![("foo".to_string(), "bar".to_string())]` results in:
```
ProviderKey:pgz EnvValues:map[foo:bar] InstanceID:c31b57c6-a564-4c28-872e-4255cf5384cf
```

Reply:
That is great!